### ISSUE TITLE
Mentor user cannot add videos

### ISSUE THREAD

**Issue by robert-dm** [hack4impact-uiuc/mentee#961]
refer to attached video

https://github.com/hack4impact-uiuc/mentee/assets/35821871/abe6c0da-b961-49c4-83c9-a8816764bc68

**Comment by robert-dm** [comment]
@Humayun0008 this issue is fixed in DEV could you please uncomment the test script?
thanks
<img width="1700" alt="image" src="https://github.com/hack4impact-uiuc/mentee/assets/35821871/3e0f3144-def0-4738-a178-aee4881d118a">

**Comment by Humayun0008** [comment]
Still not Fixed ... Please verify it again

https://github.com/hack4impact-uiuc/mentee/assets/131605337/fd58464c-033d-4cdf-89d4-da06bfe6ee2e

**Comment by robert-dm** [comment]
The problem was that the user was removed from database, or email updated to testEmail@gmail.com and I have fixed the user in the database, now it is working fine.  Attached video.   

We should not have any other problem from now on, if yes, then there is a process in your local files or anywhere in automated tests affecting the robertmurer+mentor1@gmail.com user in database.

https://github.com/hack4impact-uiuc/mentee/assets/35821871/4258bbc7-8ab0-463e-8c21-658d4e64e08a
